lib/gui/models/store.js

```diff
@@ -75,8 +75,22 @@ const ACTIONS = _.fromPairs(_.map([
   return [ message, message ];
 }));
 
-const storeReducer = (state, action) => {
-  state = state || DEFAULT_STATE;
+/**
+ * @summary The redux store reduced
+ * @function
+ * @private
+ *
+ * @param {Object} currentState - application state
+ * @param {Object} action - dispatched action
+ * @returns {Object} new application state
+ *
+ * @example
+ * const newState = storeReducer(DEFAULT_STATE, {
+ *   type: ACTIONS.REMOVE_DRIVE
+ * });
+ */
+const storeReducer = (currentState, action) => {
```

Comment:
Oh, fixed! For some reason I though I could only use a default value for the last argument (?)